app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -75,55 +84,62 @@ public void unbind() {
         }
     };
 
-    private void setClearSearchVisible(@NonNull final Boolean clearVisible) {
-        Preconditions.checkNotNull(closeButton, "Close Button has not been bound");
+    private void setClearSearchVisible(final boolean clearVisible) {
+        checkNotNull(closeButton, "Close Button has not been bound");
 
         get(closeButton).setVisibility(clearVisible ? View.VISIBLE : View.GONE);
     }
 
     public static void open(@NonNull final Context context) {
         checkNotNull(context, "Launching context cannot be null");
+
         Intent intent = new Intent(context, SearchActivity.class);
         context.startActivity(intent);
     }
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_search);
+        setContentView(layout.activity_search);
```

Comment:
Because of AS recommendations, the majority of R are from our resources, if we would be using from somewhere else, then we would use it the full package name from those resources